daemon/cluster/cluster.go

```diff
@@ -104,6 +104,9 @@ type Config struct {
 
 	// path to store runtime state, such as the swarm control socket
 	RuntimeRoot string
+
+	// EventChannel is used to push cluster events to daemon
+	EventCh chan *swarmapi.WatchMessage
```

Comment:
Comment doesn't match the field name.

Maybe this can just be `Events`? I like to avoid putting type information in the field names.